### ISSUE TITLE
amenthes iso image hosted on GitHub Pages 

### ISSUE THREAD

**Issue by ghost** [yuvadm/amenthes#9]
Hi @yuvadm,

what do you think about creating an extra GitHub Pages, see [GitHub Pages](https://pages.github.com/) for hosting the Amenthes iso image?

I would be great to provide a built iso image of amenthes (as the latest CI build is currently not available) - i saw some repositories hosting built versions of their projects, but I'm not 100% sure if we can upload a ~ 300 mb iso file.

On this page we also could provide a kind of checksum (anything but md5) to verifiy the integrity of the (downloaded) image - this possibility is still missing.

Thanks + regards,

Stefan

**Comment by yuvadm** [comment]
@stefan-it yeah I took the CI server down and didn't have a chance to provision a new one. The only reason why not to do this is because that would bloat the entire repo, and I'm not keen on doing that.

Probably a better way would be to use the [releases API](https://github.com/blog/1547-release-your-software) where we can upload binaries for testing purposes. This needs to be clearly marked as such, though. I'd be happy to make you a collaborator on the repo if you'd like to take care of this :)

**Comment by ghost** [comment]
Totally forgot the possibility to provide releases on GitHub :) -> I'd really like to be a collaborator for this repo - Thanks!

**Comment by yuvadm** [comment]
Added!
